pages/common/tailscale.md

```diff
@@ -0,0 +1,36 @@
+# tailscale
+
+> A private WireGuard network service.
+> More information: <https://tailscale.com>.
+
+- Connect to Tailscale:
+
+`sudo tailscale up`
+
+- Disconnect from Tailscale:
+
+`sudo tailscale down`
+
+- Display the current Tailscale IP addresses:
+
+`tailscale ip`
+
+- Ping a peer node at the Tailscale layer and display which route it took for each response:
+
+`tailscale ping {{ip|hostname}}`
+
+- Analyze the local network conditions and display the result:
+
+`tailscale netcheck`
+
+- Start a web server for controlling Tailscale:
+
+`tailscale web`
+
+- Display a shareable identifier to help diagnose issues:
```

Comment:
Identifier? That sounds like an ID number. Does this upload a diagnostic report to a remote server or something and return the id?

Reply:
from <https://tailscale.com/kb/1080/cli/#bugreport>
> Bugreport makes it easier to report bugs to the Tailscale team by marking diagnostic logs with indicators to make triage easier.
> 
> If you encounter a connectivity issue, run tailscale bugreport on the device experiencing the issue at the time you encounter it. This command will print a random identifier into diagnostic logs, which you can share with our team.
> 
> Identifiers look like this:
> 
> $ tailscale bugreport
> BUG-1b7641a16971a9cd75822c0ed8043fee70ae88cf05c52981dc220eb96a5c49a8-20210427151443Z-fbcd4fd3a4b7ad94
> 
> This command shares no personally-identifiable information, and is unused unless you share the bug identifier with our team.

Reply:
Ah, I see! Thanks for explaining - that makes sense now :-)